apps/docs/stories/components/Summary/Docs.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta, Title, ArgTypes, Source } from '@storybook/blocks';
+import { filterDocsByTag, ExportedParts } from '../../utils';
+import { codeExampleFull } from './example';
+import { ComponentBox } from '../../utils/documentation-components';
+import * as JustifiCheckoutSummary from './index.stories';
+import { setUpMocks } from '../../utils/mockAllServices';
+
+{setUpMocks()}
+
+<Meta
+  title="Modular Checkout/Checkout Summary"
+  Components="justifi-checkout-summary"
+  of={JustifiCheckoutSummary}
+/>
+
+<Title />
+
+---
+
+Renders a summary of checkout details as part of a checkout flow.
```

Comment:
Update to `Renders a summary of checkout details consisting of the checkout `description` and `amount` passed in the create checkout API request as part of a checkout flow.'

Make create checkout API  a link to https://docs.justifi.tech/api-spec#tag/Checkouts/operation/CreateCheckout